modules/nw-ingress-controller-nodeportservice-projects.adoc

```diff
@@ -0,0 +1,161 @@
+    namespaceSelector:
+     matchLabels:
+       <key>: <value> <3>
+    endpointPublishingStrategy:
+      type: NodePortService 
+kind: List
+metadata:
+  resourceVersion: ""
+  selfLink: ""
+# ...
+----
+<1> Specify the a custom `name` for the `IngressController` CR.
+<2> The DNS name that the Ingress Controller services.
+<3> Specify the label for a set of namespaces. Substitute `<key>:<value>` with a map of key-value pairs where `<key>` is a unique name for the new label and `<value>` is its value. For example: `ingresscontroller: custom-ic`.
+
+. Add a label to a node by using the `oc label node` command:
++
+[source,terminal]
+----
+$ oc label node <node_name> <key>=<ingress_controller_custom_resource> <1>
+----
+<1> Where `<ingress_controller_custom_resource>` must match the key-value pair that you specified in the `nodePlacement` section of your `IngressController` CR.
+
+. Create the `IngressController` object:
++
+[source,terminal]
+----
+$ oc create -f <ingress_controller_cr>.yaml
+----
+
+. Find the port for the service that was created for the `IngressController` CR:
++
+[source,terminal]
+----
+$ oc get svc -n openshift-ingress
+----
++
+.Example output that shows port `80:32432/TCP` for the `router-nodeport-custom-ic3` service
+[source,terminal]
+----
+NAME                        TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                                     AGE
+router-internal-default     ClusterIP   172.30.195.74    <none>        80/TCP,443/TCP,1936/TCP                     223d
+router-nodeport-custom-ic3   NodePort    172.30.109.219   <none>        80:32432/TCP,443:31366/TCP,1936:30499/TCP   155m
+----
+
+. Use the `oc new-project` command to create a new project:
++
+[source,terminal]
+----
+$ oc new-project <project_name>
+----
+
+. Enter the following command to label the new namespace:
++
+[source,terminal]
+----
+$ oc label namespace <project_name> <key>=<value> <1>
+----
+<1> Where `<key>=<value>` must match the value that you specified in the `namespaceSelector` section of your Ingress Controller CR.
+
+. Create a new application in your cluster:
++
+[source,terminal]
+----
+$ oc new-app --image=<image_name> <1>
+----
+<1> An example of `<image_name>` is `quay.io/openshifttest/hello-openshift:multiarch`.
+
+. Create a `Route` object for a service, so that the pod can use the service to expose the application external to the cluster.
++
+[source,terminal]
+----
+$ oc expose svc/<service_name> --hostname=<svc_name>.<project_name>.<domain_name> <1>
+----
++
+[NOTE]
+====
+You must specify the hostname and domain of your custom Ingress Controller in the `--hostname` argument. If you do not do this, the Ingress Operator uses the default Ingress Controller to serve all the routes for your cluster. 
+====
+
+. Update the `IngressController` CR to prevent the default Ingress Controller from managing the `NodePort`-type `Service`. The default Ingress Controller will continue to monitor all other cluster traffic.
```

Comment:
```suggestion
. Update the default `IngressController` CR to prevent the default Ingress Controller from managing the `NodePort`-type `Service`. The default Ingress Controller will continue to monitor all other cluster traffic.
```